lib/perl/Genome/Model/ClinSeq/Command/Converge/Base.pm

```diff
@@ -483,7 +483,7 @@ sub getModelsBuilds{
     my @models = $mg->models;
     $target_count = scalar(@models);
   }
-  if ($verbose){Genome::Model->debug_message("\nSearching for $target_count models/builds");}
+  if ($verbose){$self->debug_message("\nSearching for $target_count models/builds");}
```

Comment:
The newlines that @tmooney pointed out occur in many places not just the specific one he mentioned.